Podfile

```diff
@@ -142,7 +142,7 @@ abstract_target 'Apps' do
 
   pod 'NSURL+IDN', '~> 0.4'
 
-  pod 'WPMediaPicker', '~> 1.8', '>= 1.8.10-beta.1'
+  pod 'WPMediaPicker', '~> 1.8', '>= 1.8.10'
```

Comment:
Now that the stable version for 1.8.10 has been released, there's no need to set the beta as the minimum compatible one.